### DISCUSSION TITLE
Format of bed file CallPeaks with paired end data

### DISCUSSION THREAD

**Discussion by danielcgingerich** [stuart-lab/signac#557] [Q&A] [answered]
From the source code of `CallPeaks()`: 
```
cmd <- paste0(
    macs2.path,
    " callpeak -t ",
    object,
    " -g ",
    as.character(x = effective.genome.size),
    broadstring,
    " -f BED --nomodel --extsize ",
    as.character(x = extsize),
    " --shift ",
    as.character(x = shift),
    " -n ",
    as.character(x = name),
    " --outdir ",
    outdir,
    " ",
    additional.args
  )
```

Should the format (-f) be BEDPE instead of just BED?  10X scATAC data are paired end reads, right?  I saw this comment on biostars: 
"Regarding the case of when you have paired-end data, the following is mentioned in MACS2 manual:
'BAM/SAM If the format is BAM/SAM, please check the definition in (http://samtools.sourceforge.net/samtools.shtml). If the BAM file is generated for paired-end data, MACS will only keep the left mate(5' end) tag. However, when format BAMPE is specified, MACS will use the real fragments inferred from alignment results for reads pileup.
BEDPE or BAMPE A special mode will be triggered while format is specified as 'BAMPE' or 'BEDPE'. In this way, MACS2 will process the BAM or BED files as paired-end data. Instead of building bimodal distribution of plus and minus strand reads to predict fragment size, MACS2 will use actual insert sizes of pairs of reads to build fragment pileup.'"

**Comment by timoast** [comment, marked as answer]
The scATAC-seq reads are paired end, but the fragment file is not, and we use the fragment file to call peaks. See this thread for a bit more explanation: https://twitter.com/XiChenUoM/status/1336658454866325506
